### ISSUE TITLE
Is there a way to tell RenderedCompactJsonFormatter or to not render some placeholders?

### ISSUE THREAD

**Issue by james-hu** [serilog/serilog-formatting-compact#68]
I have this use case:

- There are multiple variables I want to included in logging
- For most of them, they are strings or numbers, so that I am happy for them to be rendered in the message
- Some of them are huge objects, so that I want it to be not rendered in the message, but still attached as a destructured property of the output

This is the example code with "$" showing the use case:
```
_logger.LogDebug("Created a Job {jobId} for {customer}. Details: {$job}", jobId, customerName, jobDetails);
```
For CompactJsonFormatter and RenderedCompactJsonFormatter, the output does not contain destructured details of jobDetails in the property "job". So that it is not very useful to me.

This is the example code with "@" showing the use case:
```
_logger.LogDebug("Created a Job {jobId} for {customer}. Details: {@jobDetails}", jobId, customerName, jobDetails);
```

For both CompactJsonFormatter and RenderedCompactJsonFormatter, the output contains destructured details of jobDetails in the property "job".
For RenderedCompactJsonFormatter, the rendered message contains fully destructured details of jobDetails too, so that the same information gets duplicated. Is there a way to tell  RenderedCompactJsonFormatter not to render it in the message?

**Comment by nblumhardt** [comment]
This should be closer to what you want:

```csharp
_logger
    .ForContext(LogEventLevel.Debug, "jobDetails", jobDetails, destructureObjects: true)
    .Debug("Created a Job {jobId} for {customer}. Details: {$job}", jobId, customerName, jobDetails);
```

Edit: this uses the Serilog ILogger API; should be adaptable to MEL's ILogger with a bit more code.

**Comment by james-hu** [comment]
Yes, this should work, I didn't think of this approach ☺️ Thanks!
Based on this approach, I think an extension can be created as a sugar for making the code easier to write.
